ext/settings.html

```diff
@@ -3213,6 +3221,87 @@ <h1>Yomitan Settings</h1>
     </div>
 </div></div>
 
+<!-- Generate anki deck modal -->
+<div id="generate-anki-deck-modal" class="modal" tabindex="-1" role="dialog" hidden><div class="modal-content modal-content-full">
+    <div class="modal-header"><div class="modal-title">Anki Deck Generator</div></div>
+    <div class="modal-body generate-anki-deck-layout">
+        <div class="generate-anki-deck-info">
+            <p class="warning-text">
+                WARNING: This feature is experimental!
+            </p>
+            <p>
+                Enter a newline separated list of terms below to send directly to an Anki deck or export to an Anki deck file in <code>Notes in plain text (.txt)</code> format.
+            </p>
+        </div>
+        <textarea autocomplete="off" spellcheck="false" id="generate-anki-deck-textarea" class="no-wrap margin-above" data-tab-action="indent,4"></textarea>
+        <div class="generate-anki-deck-test-container margin-above">
+            <p>
+                Active Anki deck: <code id="generate-anki-deck-active-deck"></code><br>
+                Active Anki card format: <code id="generate-anki-deck-active-card-format"></code>
```

Comment:
We use term `Model` in the Anki configuration, is this confusing?: 
<img width="587" alt="image" src="https://github.com/themoeway/yomitan/assets/2382537/a585108e-e813-41f5-9866-208236015da4">

Reply:
I'll rename this to `Model` for consistency since the exact setting it pulls from is `Model` but there are other Yomitan settings that use `Card` for this instead. Maybe those should be fixed too.